tests/integration/test_question_pages.py

```diff
@@ -11,8 +13,26 @@ def test_get_question_summary_page(django_app):
     question = section.question_set.first()
     answer = question.answer_set.first()
     question_summary_url = f"/consultations/{consultation.slug}/sections/{section.slug}/questions/{question.slug}"
+    question_page = django_app.get(question_summary_url)
+    page_content = html.unescape(str(question_page.content))
+
+    assert question.text in page_content
+    assert answer.theme.summary in page_content
 
     question_page = django_app.get(question_summary_url)
+    assert question.text in page_content
+    assert answer.theme.summary in page_content
+
+    for keyword in answer.theme.keywords:
+        assert keyword in page_content
+
+    if question.multiple_choice_options:
```

Comment:
I think better to explicitly create a multiple choice question rather than rely on chance — let's test the most complicated case.

See if you can update the `QuestionFactory` to have a `multiple_choice` argument as well as a `with_answer` argument?

Reply:
That's now working - thanks for your help!